### PR TITLE
Made some edits suggested by @tomatopeel in #10 and #11

### PR DESCRIPTION
**Pull request by ttmc**
I wasn't able to push changes to pull requests #10 or #11 (from @tomatopeel), so I made some changes to the Primer based on those two pull requests, and those changes are what is in _this_ pull request.

### CODE REVIEW

**Comment by ttmc** [comment]
@tomatopeel and @kremalicious: I will leave this pull request open for a day or so, to give you a chance to comment on it (if you care to do so). Then I will merge it.

File: src/primer.tex [hunk 1]
```diff
@@ -124,7 +124,7 @@ \section{Bitcoin and Other Decentralized Protocols}
 The core Internet protocols are decentralized.
 Many other Internet-based protocols are similarly decentralized,
```
Comment (tomatopeel) [inline src/primer.tex]: In my mind both Internet and internet are valid dependent upon context. If you're referring to either "The Internet" in the sense of the high-level public network accessible on the public IP ranges, then I'd use a capital 'I'. If you're referring to internetworking in general, which seems to be the case here (I read it as internet[working] protocols) then I'd use a lower-case 'i'. If referring to Internet Protocol (IP) the standard/specification then I'd capitalize.

Comment (ttmc) [reply src/primer.tex]: That makes sense. I will change those instances.

Comment (kremalicious) [reply src/primer.tex]: well, @carlysheridan says lowercase, so lowercase it is. Consistency is the important thing here

**Comment by kremalicious** [comment]
thanks @tomatopeel & @ttmc, much appreciated. Internally, we follow the AP Styleguide for all our content and they [changed their rule on "internet" last year](https://twitter.com/apstylebook/status/716279065888563200), now lowercasing it everywhere. So for consistency we just follow that rule

**Pull request merged**